cinder/base.html

```diff
@@ -75,11 +79,13 @@
     </div>
 
     <footer class="col-md-12 text-center">
+        {% block footer %}
         <hr>
         <p>{% if config.copyright %}
         <small>{{ config.copyright }}<br></small>
         {% endif %}
         <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a>.</p></small>
```

Comment:
Thoughts about removing this MkDocs link and replacing it with reasonable project-specific default text that might  be acceptable across many/most projects?

Reply:
I think it should be kept. From user standpoint - the footer is small enough that it would be very easy to overwrite if needed. 

It should also be kept to give credit to MkDocs. If anything:

```suggestion
<small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a> 
with the <a href="https://sourcefoundry.org/cinder/">cinder theme</a>.</small>
```

Reply:
I have this in my personal main.html, I can see this being useful for other projects as well:

```suggestion
        <small>Documentation built with <a href="http://www.mkdocs.org/">MkDocs</a> and    <a href="https://sourcefoundry.org/cinder/">Cinder theme.</a></small>
{% if page.meta.revision_date %}
<small><br>Revised on: {{ page.meta.revision_date }}</small>
{% endif %}
```

Reply:
Sounds good.  Will leave the MkDocs link in the footer.  +/- on the theme link on this end.  Feel free to commit it if you'd like to add it.   Will get this merged over the weekend and push a new release to Github releases and PyPI.  Thanks again for making these changes!

Reply:
I feel the same about the theme link, it looks tacky let's leave it out. No problem!

Reply:
Wasn't a criticism of your choice to add it!  I love that people want to provide acknowledgments.  I am just not a huge fan of it being a default enforced by a project author team.